### PR TITLE
[Docker] Add ARM64 support to chip-build image

### PR DESCRIPTION
**Pull request by ATmobica**
- Adapt chip-build to build on ARM64.
- Allow pytest installation in the Pigweed environment on ARM64.

### CODE REVIEW

File: integrations/docker/images/chip-build/Dockerfile [hunk 1]
```diff
@@ -184,14 +232,30 @@ RUN set -x \
...
-    && tar xfvJ node-$CHIP_NODE_VERSION-linux-x64.tar.xz \
-    && mv node-$CHIP_NODE_VERSION-linux-x64 /opt/ \
-    && ln -s /opt/node-$CHIP_NODE_VERSION-linux-x64 /opt/node \
-    && ln -s /opt/node/bin/* /usr/bin \
-    && cd .. \
-    && rm -rf node_js \
-    && : # last line
+RUN case ${TARGETPLATFORM} in \
+        "linux/amd64") \
+            set -x \
```
Comment (bzbarsky-apple) [inline integrations/docker/images/chip-build/Dockerfile:237]: I wish there were a way to reduce the duplication here by storing the tarball name in a variable.  Is there?  Similar for the bits above.

**Review by bzbarsky-apple — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]
@ATmobica we will need a maintainer to push arm64 images as well in this case. I am not setup to do so and neiter is the automated bot.

**Comment by iMonZ** [comment]
Any news here?

**Comment by andy31415** [comment]
@ATmobica need some conflicts resolved before this can be merged

**Pull request merged**